src/Core/Services/RestService.cs

```diff
@@ -334,7 +334,7 @@ private bool TryGetStoredProcedureRESTVerbs(string entityName, [NotNullWhen(true
             {
                 if (runtimeConfig.Entities.TryGetValue(entityName, out Entity? entity))
                 {
-                    SupportedHttpVerb[] methods = entity.Rest.Methods;
+                    SupportedHttpVerb[] methods = (entity.Rest.Methods is not null) ? entity.Rest.Methods : Array.Empty<SupportedHttpVerb>();
```

Comment:
Same question as https://github.com/Azure/data-api-builder/pull/1607/files#r1277992470

Reply:
Please refer to my response in that comment.